### PR TITLE
Add home content 

### PR DESCRIPTION
**Pull request by BaptisteLegat**

### CODE REVIEW

File: public/bg.png
Comment (Dercraker) [inline public/bg.png:1]: Preferably svg and full name for file

File: src/layout/header/HeaderMegaMenu.module.css
Comment (Dercraker) [inline src/layout/header/HeaderMegaMenu.module.css:1]: We use scss not css, don't use css files

File: src/pages/landing/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      <Group mb="xs">
+        {icon}
+        <Text>{title}</Text>
+      </Group>
+      <Text size="sm" c="dimmed">
+        {description}
+      </Text>
+    </Card>
+  );
+};
```
Comment (Dercraker) [inline src/pages/landing/Content.tsx]: One component by file not more. Move it in Component/DestinationCard.tsx

File: src/pages/landing/Content.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+}) => {
+  const { hovered, ref } = useHover();
+  return (
+    <Card
+      shadow="sm"
+      padding="lg"
+      radius="md"
+      withBorder
+      ref={ref}
+      style={{ border: hovered ? '2px solid teal' : '2px solid transparent' }}>
```
Comment (Dercraker) [inline src/pages/landing/Content.tsx]: For this use hover with scss

File: src/pages/landing/HeroImageBackground.module.css
Comment (Dercraker) [inline src/pages/landing/HeroImageBackground.module.css:1]: We use scss not css, don't use css files

File: src/pages/landing/LandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { HeroImageBackground } from './HeroHeader';
+import { Content } from './Content';
+
+const LandingPage = () => {
+  return (
+    <div>
+      <HeroImageBackground />
+      <Content />
+    </div>
```
Comment (Dercraker) [inline src/pages/landing/LandingPage.tsx]: use artifact if div is not usefull

`
<>
      <HeroImageBackground />
      <Content />
</>
`

**Review by Dercraker — CHANGES_REQUESTED** [review]

**Comment by Dercraker** [comment]
you commit message must respect [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

File: src/pages/landing/LandingPage.tsx [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
-	return (
-		<div>
-			<h1>home page</h1>
-		</div>
-	);
+  return (
+    <div>
+      <HeroImageBackground />
+      <Content />
+    </div>
```
Comment (BaptisteLegat) [inline src/pages/landing/LandingPage.tsx]:
```suggestion
    <>
      <HeroImageBackground />
      <Content />
    </>
```

File: src/_variables.scss [hunk 5]
```diff
@@ -1,3 +1,5 @@
 $primary-color: #f5f5f5;
 $secondary-color: #272727;
-$tertiaire-color: #ceaa37;
+$tertiaire-color: #12b886;
+
+
```
Comment (Dercraker) [inline src/_variables.scss]: Useless Space

File: src/pages/login/LoginPage.module.css
Comment (Dercraker) [inline src/pages/login/LoginPage.module.css:1]: Not Css files

File: src/pages/login/LoginPage.tsx [hunk 6]
```diff
@@ -98,7 +98,7 @@ export function LoginPage() {
           <Anchor<'a'>
             href="#"
             fw={700}
-            onClick={event => event.preventDefault()}>
+            onClick={(event) => event.preventDefault()}>
```
Comment (Dercraker) [inline src/pages/login/LoginPage.tsx]: we are in typescript add type of event

**Review by Dercraker — CHANGES_REQUESTED** [review]

**Review by Dercraker — APPROVED** [review]

**Pull request merged**